ui/component/settingWalletServer/index.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+import { DAEMON_SETTINGS } from 'lbry-redux';
+import { doSetDaemonSetting, doClearDaemonSetting, doGetDaemonStatus, doCacheCustomWalletServers, doFetchDaemonSettings } from 'redux/actions/settings';
+import { selectDaemonSettings, selectCachedWalletServers, makeSelectSharedPrefsForKey } from 'redux/selectors/settings';
+import SettingWalletServer from './view';
+
+const select = state => ({
+  daemonSettings: selectDaemonSettings(state),
+  customServers: selectCachedWalletServers(state),
```

Comment:
customWalletServers

Reply:
Changed the selector to selectSavedWalletServers as well.